### DISCUSSION TITLE
Welcome to Web-Development-Mini-Projects Discussions!

### DISCUSSION THREAD

**Discussion by Adithya-Rama** [Adithya-Rama/Web-Development-Mini-Projects#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Adithya-Rama** [comment]
Yo Collaborators @Shashanka-rao @Sharanbr19 @KarthikNaik-SCEM ... Discussion is a Way to Communicate with each other regarding the ongoing project and discuss about any issues faced.. so that we collaborators can reach out to one another and solve it then and there... Hope we make use of this Powerful Feature... Happy Coding Everyone!!! :-)

**Comment by Adithya-Rama** [comment]
Yo Boys @Shashanka-rao @Sharanbr19 @KarthikNaik-SCEM .. Who all r Gonna Get Their Laptops Tom ?!!

**Reply by KarthikNaik-SCEM** [reply]
Sorry my dad gave laptop to laptop center 😅😅

**Reply by AdithyaRIyer** [reply]
https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.redbubble.com%2Fshop%2Fdoremon%2Bstickers&psig=AOvVaw0RS6J20fQ4cN0Xu1OxoffL&ust=1683616784806000&source=images&cd=vfe&ved=0CA4QjRxqFwoTCMDN-amX5f4CFQAAAAAdAAAAABAD

Hello EveryOne
